### ISSUE TITLE
[HANDS-ON BUG] Two different variables for one value 

### ISSUE THREAD

**Issue by ergh0** [huggingface/deep-rl-class#483]
# Describe the bug

In unit 2 in the section "Monte Carlo vs Temporal Difference Learning" the learning rate is discussed. It's denoted 
lr (in the text) as well as alpha (in the pictures/slides). I'd recommend to use the same symbol which makes it easier to understand.

**Comment by simoninithomas** [comment]
Hey there 👋 , I understand the confusion. But usually in code people use lr or learning_rate that's why we goes with it contrary to mathematical formulas.
For instance: https://stable-baselines3.readthedocs.io/en/master/modules/base.html#stable_baselines3.common.base_class.BaseAlgorithm
